fastcov.py

```diff
@@ -955,6 +983,9 @@ def parseArgs():
     parser.add_argument('-p', '--dump-statistic', dest="dump_statistic", action="store_true", help="Dump total statistic at the end")
     parser.add_argument('-v', '--version', action="version", version='%(prog)s {version}'.format(version=__version__), help="Show program's version number and exit")
 
+    parser.add_argument('-gps', '--gcov_prefix_strip', dest="gcov_prefix_strip", action="store", default=0, type=int, help="The number of initial directory names to strip off the absolute paths in the object file.")
+    parser.add_argument('-gp', '--gcov_prefix', dest="gcov_prefix", action="store", default="", help="The prefix to add to the paths in the object file.")
```

Comment:
I believe this should always be an integer?
```python
    parser.add_argument('-gp', '--gcov_prefix', dest="gcov_prefix", action="store", default=0, type=int, help="The prefix to add to the paths in the object file.")
```

Reply:
No, this is the string that is prepended to the previously stripped from the absolute paths.

Reply:
Ah, my mistake, disregard. I think `action="store"` might be superfluous though.